Documentation/network/bgp-control-plane.rst

```diff
@@ -9,6 +9,14 @@
 Cilium BGP Control Plane
 ========================
 
+BGP Control Plane provides a way for Cilium to advertise routes to Cilium-managed endpoints by
+using the `Border Gateway Protocol`_ (BGP). This allows Pod networks and/or Services to be
+reachable from outside the cluster for environments that support BGP. Since BGP Control Plane
+does not program the :ref:`datapath <ebpf_datapath>`, it should not be used to establish reachability
+within the cluster.
```

Comment:
Kudos for defining BGP on first page use. ✨

Use active voice. What does "this" refer to specifically? BGP Control Plane? Reachable by what? Clarify.
```suggestion
using the `Border Gateway Protocol`_ (BGP). BGP Control Plane allows Pod networks and/or Services to be
reachable from outside the cluster for environments that support BGP. Because BGP Control Plane
does not program the :ref:`datapath <ebpf_datapath>`, do not use it to establish reachability
within the cluster.
```